meson.build

```diff
@@ -314,6 +314,9 @@ endif
 if cc.has_header('linux/ethtool.h')
   conf.set('HAVE_ETHTOOL_H', '1')
 endif
+if cc.has_header('mtd/mtd-user.h')
+  conf.set('HAVE_MTD_USER_H', '1')
+endif
```

Comment:
I wonder if you should only compile this subsystem and device type when this header is around.  It's a lot of permanently dead code for BSD and Windows otherwise.

Reply:
It's part of the ABI if it's in libfwupdplugin (and I think it needs to be)

Reply:
Yeah I see your point.  I think then it should fail probe without that header around at least for now.

Reply:
> and I think it needs to be

Actually it doesn't need to be. I've moved it out to the plugin now, I think that's a lot cleaner.